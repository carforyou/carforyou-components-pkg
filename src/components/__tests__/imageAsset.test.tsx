import * as React from "react"
import { render } from "react-testing-library"

import { ImageAsset } from "../../"

describe("ImageAsset", () => {
  it("renders image", () => {
    const { container } = render(<ImageAsset path="kitten.jpg" />)
    expect(container).toMatchSnapshot()
  })

  it("accepts imageProps", () => {
    const { container } = render(
      <ImageAsset path="kitten.jpg" width={20} height={20} alt="Kittens" />
    )
    expect(container).toMatchSnapshot()
  })
})
