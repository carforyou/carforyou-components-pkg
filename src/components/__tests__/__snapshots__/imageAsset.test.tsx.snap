// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageAsset accepts imageProps 1`] = `
<div>
  <img
    alt="Kittens"
    height="20"
    src="test-file-stub"
    width="20"
  />
</div>
`;

exports[`ImageAsset renders image 1`] = `
<div>
  <img
    src="test-file-stub"
  />
</div>
`;
